### DISCUSSION TITLE
Unreachable log info of frame_try_runtime::TryRuntime

### DISCUSSION THREAD

**Discussion by Chralt98** [KILTprotocol/mashnet-node#361] [General]
[This](https://github.com/KILTprotocol/kilt-node/blob/develop/runtimes/spiritnet/src/lib.rs#L1216) will never get printed, if it relies on [frame-executive](https://docs.substrate.io/rustdocs/latest/src/frame_executive/lib.rs.html#265-272). Because the first unwrap in `try_runtime_upgrade()` would lead to a panic, therefore the code stops executing and wouldn't reach the `log::info` from the call stack.

```rust
#![allow(unused)]
fn main() {
  try_runtime_upgrade().map_err(|err| {
      println!("Additional information");
      err
  }).unwrap();
}

fn try_runtime_upgrade() -> Result<(), &'static str> {
    let x: Result<(), &'static str> = Err("fatal error").unwrap();
    Ok(())
}
```

```text
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "fatal error"', src/main.rs:7:58
```
"Additional information" never mentioned.

**Comment by weichweich** [comment]
Hi @Chralt98 the unreachable code was removed as part of KILTprotocol/kilt-node/pull/375.
